modules/kubernetes/aks-core/variables.tf

```diff
@@ -409,6 +409,11 @@ variable "promtail_config" {
   })
 }
 
+variable "ingress_healthz_dns_zone" {
+  description = "The ingress-healthz dns zone that should be used"
+  type        = string
+}
+
```

Comment:
Thinking about changing this to `main_dns_zone` since I need this variable in multiple locations in EKS. If so I think I should do the same in AKS.

Reply:
We are using the first item in the dns_zone list as the "main" dns zone to use.